src/utilities/prTitleParser.ts

```diff
@@ -0,0 +1,35 @@
+// TODO: (dunyakirkali) Better naming
+export const parsePRTitle = (title: string, category: string): boolean => {
+  if (category === 'MAJOR') {
+    return true;
+  }
+
+  const semVerTitleRegExp = /bump .* from (?<from>.*) to (?<to>.*)/u;
+  const match = semVerTitleRegExp.exec(title);
+
+  if (match) {
+    const semVerRegExp = /^(?<major>0|[1-9]\d*)\.(?<minor>0|[1-9]\d*)\.(?<patch>0|[1-9]\d*)$/u;
+
+    // TODO: (dunyakirkali) Handle optionals
+    const from = match.groups?.from!;
+    const to = match.groups?.to!;
+
+    // TODO: (dunyakirkali) Handle optionals
```

Comment:
TODOs should have a deadline.

Reply:
Example: `TODO (dunyakirkali) [2020-10-01]: Handle optionals.`

Reply:
so what is the convention in TS to handle these optional values?

Do I have to pepper the code with `=== undefined` ?

Reply:
Yeah, or you could default to some value. E.g. `const to = match.groups?.to ?? '';`